docs/software/ml/pytorch.md

```diff
@@ -0,0 +1,392 @@
+        | huggingface_hub     | 0.26.2 |
+        | idna                | 3.4 |
+        | importlib_metadata  | 7.0.1 |
+        | iniconfig           | 2.0.0 |
+        | itsdangerous        | 2.1.2 |
+        | Jinja2              | 3.1.4 |
+        | joblib              | 1.2.0 |
+        | lightning-utilities | 0.11.2 |
+        | MarkupSafe          | 2.1.3 |
+        | mpmath              | 1.3.0 |
+        | networkx            | 3.1 |
+        | nltk                | 3.9.1 |
+        | numcodecs           | 0.15.0 |
+        | numpy               | 2.1.2 |
+        | nvtx                | 0.2.5 |
+        | packaging           | 24.1 |
+        | pillow              | 11.0.0 |
+        | pip                 | 23.1.2 |
+        | platformdirs        | 3.10.0 |
+        | pluggy              | 1.5.0 |
+        | protobuf            | 5.28.2 |
+        | psutil              | 7.0.0 |
+        | pybind11            | 2.13.6 |
+        | pydantic            | 2.10.1 |
+        | pydantic_core       | 2.27.1 |
+        | pytest              | 8.2.1 |
+        | pytest-asyncio      | 0.23.5 |
+        | pytest-cov          | 4.0.0 |
+        | pytest-mock         | 3.10.0 |
+        | pytest-random-order | 1.0.4 |
+        | pytz                | 2023.3 |
+        | PyYAML              | 6.0.2 |
+        | regex               | 2022.8.17 |
+        | requests            | 2.32.3 |
+        | safetensors         | 0.4.5 |
+        | sentencepiece       | 0.1.99 |
+        | sentry-sdk          | 2.22.0 |
+        | setproctitle        | 1.1.10 |
+        | setuptools          | 69.2.0 |
+        | six                 | 1.16.0 |
+        | smmap               | 5.0.0 |
+        | sympy               | 1.13.1 |
+        | tiktoken            | 0.4.0 |
+        | tokenizers          | 0.21.0 |
+        | torch               | 2.6.0 |
+        | torchaudio          | 2.6.0a0+d883142 |
+        | torchmetrics        | 1.5.2 |
+        | torchvision         | 0.21.0 |
+        | tqdm                | 4.66.3 |
+        | transformer_engine  | 2.3.0.dev0+dd4c17d |
+        | transformers        | 4.48.3 |
+        | triton              | 3.2.0+gitc802bb4f |
+        | typing_extensions   | 4.12.2 |
+        | urllib3             | 2.1.0 |
+        | versioneer          | 0.29 |
+        | wandb               | 0.19.9 |
+        | Werkzeug            | 3.0.4 |
+        | wheel               | 0.41.2 |
+        | wrapt               | 1.15.0 |
+        | zarr                | 3.0.1 |
+        | zipp                | 3.17.0 |
+
+
+[](){#ref-uenv-pytorch-how-to-use}
+## How to use
+
+There are two ways to access the software provided by the uenv, once it has been started.
+
+=== "the default view"
+
+    The simplest way to get started is to use the `default` file system view,
+    which automatically loads all of the packages when the uenv is started.
+
+    !!! example "test mpi compilers and python provided by pytorch/v2.6.0"
+        ```console
```

Comment:
For this kind of comments, I think [annotations](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-annotations) are better, so that the user can focus on the actual commands and visualize the help/explainer only if needed. But this might be personal preference, so up to you if you want to change it (but some comments are a bit "obvious" and could indeed be hidden, like `start using the default view` for `uenv start --view=default`).